### DISCUSSION TITLE
模仿fidder开发个条件筛选功能

### DISCUSSION THREAD

**Discussion by hs3434** [wanghongenpin/proxypin#324] [Ideas]
调试的时候经常需要根据进程、端口，或者根据内容筛选来持续跟踪请求。
不筛选的话，请求太多，眼花缭乱

**Comment by wanghongenpin** [comment]
<img width="506" alt="image" src="https://github.com/user-attachments/assets/5fe366fc-16d7-4f1b-92e6-50aedbeb4284">
有筛选

**Reply by hs3434** [reply]
原来有啊，不好意思没注意到，谢谢，我用用看。
